src/elements/OcTextField.vue

```diff
@@ -1,18 +1,13 @@
 <template>
-  <component :is="wrapper" :class="['input', { 'input-expand': width === 'expand' }]">
-    <label :for="id" v-if="label">{{ label }}</label>
-    <input
-      :id="id"
-      :disabled="disabled"
-      :type="type"
-      :value="value"
-      :class="state"
-      :placeholder="placeholder"
-      :aria-label="placeholder"
-      @input="onInput($event.target.value)"
-      @focus="onFocus($event.target.value)"
-    />
-  </component>
+  <input
+    class="uk-input"
+    :type="type"
+    :value="value"
+    :placeholder="placeholder"
+    :aria-label="placeholder"
```

Comment:
from an accessibilty pov we should really keep the label if I understood this well ..... see #21 ...